homeassistant/components/sabnzbd/__init__.py

```diff
@@ -24,12 +24,15 @@
 
 _CONFIGURING = {}
 
+CONF_URLBASE = 'urlbase'
+
 ATTR_SPEED = 'speed'
-BASE_URL_FORMAT = '{}://{}:{}/'
+BASE_URL_FORMAT = '{}://{}:{}/{}'
```

Comment:
You don't need to build the base path into the url yourself. The pysabnzbd library accepts a `web_root` param in its initializer for this.